pkg/es/config/config.go

```diff
@@ -63,6 +63,7 @@ type Configuration struct {
 	BulkFlushInterval     time.Duration  `mapstructure:"-"`
 	IndexPrefix           string         `mapstructure:"index_prefix"`
 	IndexDateLayout       string         `mapstructure:"index_date_layout"`
+	IndexRotate           string         `mapstructure:"-"`
```

Comment:
ES uses the term `Rollover`, I think we should stay consistent with terminology.